### PR TITLE
adding copyrights and small update to doc

### PR DESCRIPTION
**Pull request by tsurdilo**
just adding copyrights and small doc update

### CODE REVIEW

**Comment by robholland** [comment]
I don't think we need the license in every file. Is there some strong reason for doing this?

File: docs/how-to-use.md [hunk 1]
```diff
@@ -25,6 +26,23 @@ cd background-checks
 
 This starts up a new Docker Application with a stack of containers.
 
+## How to clean up docker containers
+
+* Stop all containers
+```
+docker kill $(docker ps -q)
```
Comment (robholland) [inline docs/how-to-use.md:33]:
```suggestion
docker compose stop
```

`docker kill $(docker ps -q)` stops all of the docker containers on the system, regardless of they are related to the background-checks application.

File: docs/how-to-use.md [hunk 2]
```diff
@@ -25,6 +26,23 @@ cd background-checks
...
+## How to clean up docker containers
+
+* Stop all containers
+```
+docker kill $(docker ps -q) 
+```
+
+* Remove all container
+```
+docker rm $(docker ps -a -q)
```
Comment (robholland) [inline docs/how-to-use.md:38]:
```suggestion
docker compose rm
```

File: docs/how-to-use.md [hunk 3]
```diff
@@ -25,6 +26,23 @@ cd background-checks
...
+```
+
+* Remove all images
+```
+docker rmi $(docker images -q)
```
Comment (robholland) [inline docs/how-to-use.md:43]: Not sure why this is needed?

**Review by robholland — CHANGES_REQUESTED** [review]

**Comment by tsurdilo** [comment]
@robholland if you think we don't need to license header thats fine..we have them in all our samples at least for java ones so thought we could add. feel free to disregard

**Comment by tsurdilo** [comment]
resolved comments

**Comment by robholland** [comment]
samples-go doesn't have the top of file licences, so lets stick without them.

**Comment by tsurdilo** [comment]
to be honest i rather add them to the go samples than not add them here cause samples don't have it :) 
to me at least for OSS license headers are important

**Comment by robholland** [comment]
As discussed on Slack, please remove the per file license headings.

**Comment by tsurdilo** [comment]
will close this one and redo it as we talked about

**Pull request closed without merging**